components/skus/browser/rs/lib/src/sdk/credentials/mod.rs

```diff
@@ -40,6 +40,37 @@ where
         let expires_at = order.expires_at;
         for item in &order.items {
             match item.credential_type {
+                CredentialType::TimeLimitedV2 => {
+                    let expires_at = self
+                        .last_matching_time_limited_v2_credential(&item.id)
+                        .await?
+                        .map(|cred| cred.valid_to)
+                        .or(expires_at);
+                    if let Some(expires_at) = expires_at {
+                        // attempt to refresh credentials if we're within 5 days of expiry
+                        if Utc::now().naive_utc() > (expires_at - chrono::Duration::days(5)) {
+                            let refreshed = self.refresh_order_credentials(order_id).await;
+                            if refreshed.is_err() {
+                                continue;
+                            }
+                        }
+                    }
+
+                    if let Some(creds) = self.matching_time_limited_v2_credential(&item.id).await? {
+                        let unblinded_creds =
+                            creds.unblinded_creds.ok_or(InternalError::NotFound)?;
+                        let remaining_credential_count =
+                            unblinded_creds.into_iter().filter(|cred| !cred.spent).count();
+
+                        let active = remaining_credential_count > 0;
+                        return Ok(Some(CredentialSummary {
+                            order,
+                            remaining_credential_count: remaining_credential_count, // number unspent
```

Comment:
could be just `remaining_credential_count,`

Reply:
https://github.com/brave/brave-core/pull/15465/commits/9c9188c99dcc3e8bb08fd4835f602a62155975ae